static/js/featuredialog.js

```diff
@@ -0,0 +1,45 @@
+var FeatureDialog = Backbone.View.extend({
+	className: "feature-dialog",
+    events: {
+		"submit" : "submit",
+		"click .save-button" : "submit",
+		"click .close-button" : "close"
+	},
+	initialize: function(options) {
+		this.markers = options.markers;
+		this.type = options.type;
+		this.event = options.event;
+
+	},
+	render: function() {
+		this.$el.append($("#feature-dialog").html());
+		$(document.body).append(this.$el);
+		this.modal = this.$el.find(".modal");
+        this.modal.modal("show");
```

Comment:
I tried to remove the "this.modal.modal("show") but It didn't work without it. It just made the screen fade and that's it.